doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -0,0 +1,71 @@
+(kuberay-tpu)=
+
+# Using TPUs with KubeRay
+
+This document provides tips on TPU usage with KubeRay.
+
+TPUs are available on Google Kubernetes Engine (GKE). To use TPUs with Kubernetes, configure
+both your Kubernetes setup and add additional values to your RayCluster configuration.
+Instructions for configuring a RayCluster with TPUs on GKE can be found here:
+
+- {ref}`kuberay-gke-tpu-cluster-setup`
+
+## Quickstart: Serve a Stable Diffusion model on GKE with TPUs
+
+After setting up a GKE cluster with TPUs and the Ray TPU initialization webhook, you're ready to begin running
```

Comment:
<img width="1316" alt="image" src="https://github.com/user-attachments/assets/9b68c508-5430-4519-84ac-d722892b757d">

Reply:
I added in the ref to the Stable Diffusion example this way because that's how I saw other docs in the repo do it, I'm not sure how this will appear once merged (since the Stable Diffusion TPU example that's referenced will be merged in the same PR).